osf/models/node.py

```diff
@@ -1335,18 +1346,6 @@ def copy_contributors_from(self, resource):
                 self.add_permission(contrib.user, permission, save=True)
         Contributor.objects.bulk_create(contribs)
 
-    def subscribe_contributors_to_node(self):
```

Comment:
So how is this done after the refactor? Or maybe this is just dead code?

Reply:
Never used even on develop